### ISSUE TITLE
Minimum Viable Product of Jupyter Book functionality with MyST

### ISSUE THREAD

**Issue by choldgraf** [executablebooks/mystmd#1106]
We're tracking a [laundry list of MyST functionality in this issue](https://github.com/executablebooks/mystmd/issues/189). There's a lot of functionality in there, and that issue doesn't prioritize or define the essential vs. "nice to have" functionality we'd want to see.

In order to move forward [the initiative on a myst backend for Jupyter Book](#1000) , I think we should define the minimal functionality that makes "Jupyter Book via the MyST engine" safe to try. That can be the guiding principle to drive @agoose77's work once we've finished up [the launch initiative](#1001) .

### Minimal functionality needed to use Jupyter Book with MyST

Here's a starting point, we can change or update this as we wish:

### MyST engine

- [ ] Nested parsing of MyST within directives (enables things like https://github.com/executablebooks/mystmd/issues/893)
- [ ] Streamline in-line execution / variable insertion either via documentation or new functionality
- [ ] Cross-MyST site references (intersphinx or MyST)

### Documentation

- [ ] https://github.com/executablebooks/mystmd/issues/1094
- [ ] https://github.com/executablebooks/mystmd/issues/927

### Book theme

- [ ] Make URLs represent the folder structure of content files (this will allow things like multiple translation versions as an interrim solution to #166 )
- [ ] Basic functionality for external links (social, github, etc)
- [ ] In-page navigation on mobile (e.g. via right sidebar, subset of left sidebar, etc)
- [ ] https://github.com/executablebooks/mystmd/issues/100
- [ ] Fix any major bugs associated with static HTML generation ([link to bugs in myst-theme](https://github.com/executablebooks/myst-theme/issues?q=is:issue+is:open+sort:updated-desc+label:bug))

### On the fence

_Not sure if this is MVP-worthy or not_

- [ ] https://github.com/executablebooks/mystmd/issues/840 (so that we can get MVP blogging functionality out of the box...

**Comment by choldgraf** [comment]
I've updated the project board a little bit to have three columns:

- Needed for alpha: Issues that we need to resolve before we can release an alpha version of Jupyter Book 2.0. For these issues, releasing v2 without resolving them should significantly degrade the user experience so much it's not worth releasing.
- Needed for beta: Issues that are important but not critical for getting JB2 out for feedback
- Nice to have: Issues that we've explicitly decided aren't required for a JB2 launch.

**Comment by haroldcampbell** [comment]
This needs to be on product board so we won't be tracking this here.

**Comment by agoose77** [comment]
I'm closing this -- we've deemed Jupyter Book 2 safe to try by releasing it!
